ecs-agent/tmds/handlers/taskprotection/v1/handlers/handlers.go

```diff
@@ -166,7 +166,7 @@ func UpdateTaskProtectionHandler(
 			}
 			return
 		}
-		logger.Info("GetTaskProtection endpoint was called", logger.Fields{
+		logger.Info("UpdateTaskProtection endpoint was called", logger.Fields{
```

Comment:
Separate question - can we validate the request (line #175) earlier so that we can avoid unnecessary queries of line #161?